src/message/mod.rs

```diff
@@ -99,8 +99,10 @@ impl<T: producer::MessageScheme> MessageManager for RdProducerStub<T> {
     */
 
     fn is_block(&self) -> bool {
-        self.sender.is_full()
+        // https://github.com/Fluidex/dingir-exchange/issues/119
+        //self.sender.is_full()
         //self.sender.len() >= (self.sender.capacity().unwrap() as f64 * 0.9) as usize
+        self.sender.len() >= (self.sender.capacity().unwrap() - 1000)
```

Comment:
prefer using the original style (` as f64 * 0.9`  or ` as f64 * 0.5`) here.

Otherwise if we decrease ` let (sender, receiver) = crossbeam_channel::bounded(2048);` to a small number 
and forget to change `self.sender.capacity().unwrap() - 1000`, it may become a negative number

Reply:
just a temp fix. I will keep #119 open. 
Currently I have to reserve >600 space in the queue due to tick.js